metrics/lint/linter.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+syntax = "proto3";
+package linter;
+
+// The Go package name.
+option go_package = "linter";
+
+message Message{
+    string Type = 1;
+    string Message = 2;
+    string Suggestion = 3;
+    repeated string Path = 4;
```

Comment:
From the [Protocol Buffers Style Guide](https://developers.google.com/protocol-buffers/docs/style#repeated_fields), repeated fields should be plural. Since "paths" doesn't really make sense (we're describing a single path), please use "keys" for this field.